hedera-node/configuration/mainnet/application.properties

```diff
@@ -1 +1,12 @@
-
+autoRenew.targetTypes=
+contracts.knownBlockHash=055fac8eb788872957dfee45b7d40f7c94d66ea0a75ec703b993d57e441de574@33476932
+contracts.localCall.estRetBytes=4096
+hedera.allowances.isEnabled=true
+scheduling.whitelist=CryptoTransfer,ConsensusSubmitMessage,TokenBurn,TokenMint,CryptoApproveAllowance
+sigs.expandFromLastSignedState=false
+staking.fees.stakingRewardPercentage=10
+staking.maxDailyStakeRewardThPerH=6_849
```

Comment:
This property is no more needed due to `staking.perHbarRewardRate` this property, which is set correctly in default properties

Reply:
If this is no longer applicable to the `0.41.x` release then we should remove and seek approval from the managers.

Reply:
Removed.